### PR TITLE
pretrain with sklike models

### PR DESCRIPTION
**Pull request by jckhang**
## What does this PR do?

Add support function for pretrained models

## Changes

- [x] API to upload sklike model
- [x] API to set custom code via ts

## Before reviewing

- [x] Are there any linked PRs?
- [x] Did you test this PR?
- [ ] Is there any migration or seeding necessary?
- [ ] Did you add api to only one of the language (python/typescript)?

### Links:

-   xyme-backend: https://github.com/Accern/xyme-backend/pull/149

### New API added that not implemented in other languages

If your answer is yes, please also update the todo_api.txt at where the API is missing.

### CODE REVIEW

File: packages/python/accern_xyme/accern_xyme.py [hunk 1]
```diff
@@ -2375,6 +2400,32 @@ def upload_zip(self, source: Union[str, io.BytesIO]) -> List['BlobHandle']:
...
+            model_name: str,
+            maybe_classes: Optional[List[str]],
+            maybe_range: Optional[Tuple[Optional[float], Optional[float]]],
+            full_init: bool = True) -> UploadFilesResponse:
+        try:
+            if isinstance(source, str) or not hasattr(source, "read"):
+                with open(f"{source}", "rb") as fin:
+                    self._upload_file(fin, ext="pkl")
+            else:
+                self._upload_file(source, ext="pkl")
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/accern_xyme.py:2417]: where does upload_file set the filename?

File: packages/typescript/index.ts [hunk 2]
```diff
@@ -2884,6 +2891,33 @@ export class CustomCodeBlobHandle extends BlobHandle {
             },
         });
     }
+
+    public async setCustomCode(
+        func: string,
+        funcName: string
```
Comment (JosuaKrause) [inline packages/typescript/index.ts:2897]: what about imports?

Comment (jckhang) [reply packages/typescript/index.ts:2897]: see 2870 we have them

File: packages/python/accern_xyme/accern_xyme.py (hunk 1 above)
Comment (jckhang) [reply packages/python/accern_xyme/accern_xyme.py:2417]: we always save the file as 'model.pkl' at the backend

**Review by JosuaKrause — APPROVED** [review]
update description

**Pull request merged**